spec/controllers/api/v1/book_suggestion_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let!(:book_suggestion_attributes) { attributes_for(:book_suggestion) }
+      it 'creates a new book suggestion' do
+        expect do
+          post :create, params: { book_suggestion: book_suggestion_attributes }
```

Comment:
Instead of this, do `let(:request) { post :create, params: { book_suggestion: book_suggestion_attributes } }` and call it here